mcp/src/main/java/io/modelcontextprotocol/server/McpAsyncServer.java

```diff
@@ -257,6 +258,9 @@ private static class AsyncServerImpl extends McpAsyncServer {
 
 		private final ConcurrentHashMap<String, McpServerFeatures.AsyncPromptSpecification> prompts = new ConcurrentHashMap<>();
 
+		// TODO: this field is deprecated and should be remvoed together with the
+		// broadcasting loggingNotification.
+		@Deprecated
```

Comment:
There is no need to deprecate a private field

Reply:
this is more for us not to forget to delete it. But i can remove the annotation.